sections/design/database.tex

```diff
@@ -0,0 +1,47 @@
+\section{Database choice}
+The application being developed doing this project is heavily focused on scalability, it is therefore important that a database system that can handle a lot of data and which makes scalability easier is chosen.
+\\
+Furthermore our application is going to handle a variety of data forms. It will store both structured data in the form of user profiles as well as unstructured data such as pictures and videos.
+\\
+This means that a relational and non-relational database system both could work, making the choice more difficult.
+For some time it was considered to go with a non-relational database as such database system is often used with our frontend framework React, but since the team has more experience working with relational databases it was decided to go with this kind of database system.
```

Comment:
```suggestion
We considered going with a non-relational database, as such database system is often used with our frontend framework React, but since the team has more experience working with relational databases it was decided to go with this kind of database system.
```